ui/store/actions.ts

```diff
@@ -640,76 +636,6 @@ export function setCurrentCurrency(
   };
 }
 
-export function signMsg(
-  msgData: TemporaryMessageDataType['msgParams'],
-): ThunkAction<
-  Promise<TemporaryMessageDataType['msgParams']>,
-  MetaMaskReduxState,
-  unknown,
-  AnyAction
-> {
-  log.debug('action - signMsg');
-  return async (dispatch: MetaMaskReduxDispatch) => {
-    dispatch(showLoadingIndication());
-    log.debug(`actions calling background.signMessage`);
-    let newState;
-    try {
-      newState = await submitRequestToBackground<
-        MetaMaskReduxState['metamask']
-      >('signMessage', [msgData]);
-    } catch (error) {
-      logErrorWithMessage(error);
-      dispatch(displayWarning(error));
-      throw error;
-    } finally {
-      dispatch(hideLoadingIndication());
-    }
-
-    dispatch(updateMetamaskState(newState));
-    dispatch(completedTx(msgData.metamaskId));
```

Comment:
We may need to add back this action in particular or call it elsewhere as it appears to reset some state in the `app` reducer.

Reply:
I agree, even I didn't see any anomalies on manual and e2e tests, it's better to call that on sign action. I've added it all places where we resolve approvals.